### DISCUSSION TITLE
Is there any api to pull deposit amount ?

### DISCUSSION THREAD

**Discussion by baichen111** [ib-api-reloaded/ib_async#44] [General]
Hi,
Is there any ib api to request deposit amount from my account ?

**Comment by michaelzus** [comment]
don't sure what you mean by "deposit" amount,

if you are looking for available value for trade try this:

ib = IB()
...
...
...
float([v for v in ib.accountValues() if v.tag == 'CashBalance' and v.currency == 'BASE'][0].value)

**Comment by baichen111** [comment]
not CashBalance. 
It should be cash amount transferred to IBKR trading account .
